Content.Client/Options/UI/Tabs/MiscTab.xaml.cs

```diff
@@ -33,7 +33,7 @@ public MiscTab()
         var layoutEntries = new List<OptionDropDownCVar<string>.ValueOption>();
         foreach (var layout in Enum.GetValues(typeof(ScreenType)))
         {
-            layoutEntries.Add(new OptionDropDownCVar<string>.ValueOption(layout.ToString()!, layout.ToString()!));
+            layoutEntries.Add(new OptionDropDownCVar<string>.ValueOption(layout.ToString()!, Loc.GetString($"ui-options-hud-layout-{layout.ToString()!.ToLower()}")));
```

Comment:
i think it might be bad because it's hardcoded, but since it will be only two screen types it should be fine

Reply:
> i think it might be bad because it's hardcoded, but since it will be only two screen types it should be fine

that's why I did it that way